### PR TITLE
Completed path to invite users to group via CSV.

### PR DESCRIPTION
**Pull request by bportnoy**
And many other small tweaks. There's still more work to do on the groups flow but this is getting the job done - and it's a monster PR anyway.

### CODE REVIEW

**Comment by ryyann** [comment]
Jesus, bradley you weren't kidding. This thing's huge. Looks like we'll need to resolve some merge conflicts first though.

**Comment by bportnoy** [comment]
Yikes - it was good when I submitted, but I'll take a look Monday morning.

**Comment by brianpchsu** [comment]
I think we both touch package.json, main.scss and routes.jsx. Sorry, if I know your PR touches so many files, I would wait until yours get merged. I guess there is no way we can revert our previous PR, right.

File: .gitignore [hunk 1]
```diff
@@ -15,4 +15,7 @@ test/scripts/bundle.js
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
 # ignore vagrant files
-.vagrant/
\ No newline at end of file
+.vagrant/
+#Upload directory
+server/uploads/*
+!server/uploads/tmp/
```
Comment (brianpchsu) [inline .gitignore]: Is this the directory where CSV store?

**Pull request merged**